### ISSUE TITLE
Bug in spaceInsideComments rule

### ISSUE THREAD

**Issue by meherkasam** [nicklockwood/SwiftFormat#765]
I observed this behaviour in 0.45.4 and 0.46.3.

Example code:
```
class FooClass {
    /**
    Call to configure forced disabling of Bills fallback mode.
    Intended for use only in debug builds and automated tests.
     */
    func barFunc() {}
}
```
(Notice that the `*/` part of the comment is one space to the right of the baseline)

Command run:
`swiftformat test.swift --rules spaceInsideComments`

Running the command adds another space before the `*/` to give the following diff:
```
<      */
---
>       */
```

This updated file fails the formatting check in the next dry run. Running the formatting command again results in yet another space character being added in an endless loop.

Another point to note is that if `*/` lines up horizontally with `/**`, then the formatter does not complain.

**Comment by nicklockwood** [comment]
@meherkasam thanks, I'll investigate.

**Comment by nicklockwood** [comment]
@meherkasam fixed in 0.47.4
